src/App.vue

```diff
@@ -135,6 +149,10 @@ window.addEventListener('beforeunload', function (e) {
         v-if="districtStore.checkboxChecked === true"
       ></DistrictDataTooltip>
 
+      <InstallationDataTooltip
+        v-if="isDisplayDistrictCheckbox && installationsStore.canBeDisplayed"
```

Comment:
installationsStore.canBeDisplayed is currently set in only one place. when the data will be dynamically retrieved, where will it be modified?

Reply:
The boolean is only used to be able to visualize the tooltip before the integration of the dynamic part. I set it to false before merging then I will modify the display conditions of the tootip afterwards